### PR TITLE
Add decorators for running Travis tests using cron job

### PR DESCRIPTION
**Pull request by stsci-hack**
An attempt at enabling Travis to use cron jobs to run tests on this package on a regular basis.

### CODE REVIEW

**Comment by nden** [comment]
Just to clarify, the dev tests are running now on PRs but since this is a low development package it's better to have them run as a cron job every day and not rely on a PR. The cron jobs should be set on the Travis site directly.

File: .travis.yml [hunk 1]
```diff
@@ -43,10 +43,12 @@ matrix:
         # Try Astropy development version
         - python: 3.6
           env: ASTROPY_VERSION=development SETUP_CMD="test"
+            #EVENT_TYPE='cron'
```
Comment (pllim) [inline .travis.yml]: Why is it commented?

**Comment by pllim** [comment]
I don't understand. There is already one... see https://travis-ci.org/spacetelescope/stsci.tools/builds/543305063

**Comment by pllim** [comment]
In the Travis setting, `master` is set to run weekly. The real problem is that no one is checking it, because it looks like it has been failing for a while with Astropy dev.

**Comment by nden** [comment]
Whose account is linked with it? I did not get a notification about it.

**Comment by pllim** [comment]
Travis cron does not send out any notification on failure (it is the same problem with Astropy cron stuff). Someone has to remember to check it once in a while.

**Comment by jhunkeler** [comment]
The UI layout of travis' settings page made it look like a cronjob existed, but in reality nothing was configured. I went ahead and told it to run daily, but to skip the build in the event a commit was merged within 24 hours.

:santa:

**Comment by pllim** [comment]
And it looks like they might have expanded the notification settings since the last time I checked (which was quite a while ago). You can play around with this: https://docs.travis-ci.com/user/notifications/#how-is-the-build-email-receiver-determined

**Comment by pllim** [comment]
I think this PR does not solve the problem of lack of cron job notification, so I am closing it.

**Pull request closed without merging**